### ISSUE TITLE
Publishing quality attack flow diagrams

### ISSUE THREAD

**Issue by curious-attempt-bunny** [center-for-threat-informed-defense/attack-flow#42]
Related to https://github.com/center-for-threat-informed-defense/attack-flow/issues/40, how does the community use the attack flow JSON files to publish quality diagrams, such as this one?

Are you able to share what tooling was used [to generate this]?
Edit (opening question is still valid): Found the answer. They were manually(?) generated drawio diagrams (e.g. https://github.com/center-for-threat-informed-defense/attack-flow/blob/main/data/action-object-tesla.drawio).

![image](https://user-images.githubusercontent.com/51608/168895645-c0ef8806-e379-4418-9dc1-a86870e11713.png)

**Comment by curious-attempt-bunny** [comment]
For reference, here's what the attack-flow-example looks like via the current graphviz script:

```
cd ~/Downloads
curl https://raw.githubusercontent.com/center-for-threat-informed-defense/attack-flow/main/schema/attack-flow-example.json -O -L
cd ~/Downloads/attack-flow/src
python3 -m attack_flow.scripts.graphviz ~/Downloads/attack-flow-example.json ~/Downloads/attack-flow-example.dot
dot ~/Downloads/attack-flow-example.dot -Tpng -O
```
![attack-flow-example dot](https://user-images.githubusercontent.com/51608/168897332-4646d65a-5f1d-4dc4-908c-4ff1915015fe.png)

**Comment by lcarson-2002** [comment]
So here's the "Agent Tesla" sample output from the original graphviz.py, included in the 1.0.0 release

![Screen Shot 2022-05-18 at 10 43 32 AM](https://user-images.githubusercontent.com/54375416/169108218-7652b743-5568-42cb-858f-f4d34623f664.png)
_____________________
Then after I changed the Graphviz.py (in post #40) to use the Asset State for the label, I get this output, which is arguably more usable...
![Screen Shot 2022-05-18 at 10 49 05 AM](https://user-images.githubusercontent.com/54375416/169109110-32925293-ca6c-423f-a0a4-29588257aafa.png)

-----------------------------

Finally, I see no value in the "http://flow-1" Oval at the top, so I manually edit the DOT file and comment out all of the flows from that oval.
![Screen Shot 2022-05-18 at 10 55 28 AM](https://user-images.githubusercontent.com/54375416/169110430-aaa46e99-c649-4691-957e-bb66eeb9b22a.png)

________________________
Here's my current version of graphviz.py
[graphviz.py.txt](https://github.com/center-for-threat-informed-defense/attack-flow/files/8719046/graphviz.py.txt)
